gallery/src/jsUtilPages/VulnerabilityLevelUtils/VulnerabilityLevelUtilsPage.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxP, NxCode, NxList } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const formatVulnerabilityScoreExampleCode = require('./FormatVulnerabilityScoreExample?raw'),
+    getVulnerabilitySeverityLevelExampleCode = require('./GetVulnerabilitySeverityLevelExample?raw');
+
+const VulnerabilityLevelUtilsPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        The React Shared Components library includes several simple JavaScript
+        objects and TypeScript types that provide code definitions and mappings to help represent
+        CVSS scores and severity levels.
+      </NxP>
+      <NxList bulleted>
+        <NxList.Item>
+          <NxList.Text>
+            <strong>Vulnerability (CVSS) Score:</strong>{' '}
+            A number with one decimal point precision between 0.0 and 10.0, inclusive,
+            that describes the severity of a threat. Zero represents no threat at all,
+            and ten represents the most severe of threats.
+          </NxList.Text>
+        </NxList.Item>
+        <NxList.Item>
+          <NxList.Text>
+            <strong>Vulnerability Severity Level</strong>{' '}
+            In increasing order of severity: <q>none</q>; <q>low</q>; <q>medium</q>; <q>high</q>;
+            and <q>critical</q>. Each vulnerability severity level is a member of a corresponding
+            range of CVSS Scores:
+
+            <NxList className="nx-list--bulleted">
+              <NxList.Item>
+                <NxList.Text><strong>none:</strong> 0.0</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>low:</strong> 0.1{'\u2014'}3.9</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>medium:</strong> 4.0{'\u2014'}6.9</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>high:</strong> 7.0{'\u2014'}8.9</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>critical:</strong> 9.0{'\u2014'}10.0</NxList.Text>
```

Comment:
Pretty sure emdashes don't need to be escaped here either

Reply:
Solved in https://github.com/sonatype/sonatype-react-shared-components/commit/850e0f97fb067d53160d6492597a98ad374cc162